src/styles/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $padding-size;
+  gap: $gap-size;
+  width: calc(7 * #{$day-size} + 6 * #{$gap-size} + 2 * #{$padding-size});
+  box-sizing: border-box;
+
+  &__day {
+    width: $day-size;
+    height: $day-size;
+    background-color: $day-color;
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    transition-duration: 500ms;
+
+    &::before {
+      font-family: Arial, sans-serif;
+      font-size: 30px;
+      content: '';
```

Comment:
The `content` property should not be empty. If you want to generate content in CSS, ensure it has a value or remove the pseudo-element if it's not needed.